packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -1420,3 +1453,46 @@ class ChartsValues {
     results[adbGraphicsJsonName] = androidData.graphics;
   }
 }
+
+class MemoryConfigurationsDialog extends StatelessWidget {
+  const MemoryConfigurationsDialog(this.controller);
+
+  static const dialogWidth = 700.0;
```

Comment:
does this match the other settings dialogs? we should use a consistent width. If there isn't already a defaultDialogWidth const, feel free to create one and place it where our other default const values live

Reply:
Yes it matches exactly, although I'm not convinced I need that size.  But a common is probably a good idea.